docs/999-big-list-of-options.md

```diff
@@ -1331,6 +1331,23 @@ export default {
 // -> def(['dependency'],...
 ```
 
+**output.amd.keepExtension**<br> Type: `boolean`<br> CLI: `--amd.keepExtension`<br> Default: `false`
+
+Add `.js` extension for generated chunks and local AMD modules:
```

Comment:
```suggestion
Add `.js` extension for imports of generated chunks and local AMD modules:
```

to make clear we are not changing the names of the actually generated files?

Reply:
> to make clear we are not changing the names of the actually generated files?

No, only add extension if needed.